plasmapy/formulary/tests/test_collisions.py

```diff
@@ -1698,3 +1700,347 @@ def test_kwarg_method_error(self):
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+            ),
+        ],
+    )
+    def test_limit_values(
+        self,
+        interaction_type,
+        limit_type,
+        constructor_arguments,
+        constructor_keyword_arguments,
+    ):
+        """Test the return values"""
+
+        value_test_case = CollisionFrequencies(
+            *constructor_arguments, **constructor_keyword_arguments
+        )
+
+        coulomb_density_constant = (
+            constructor_keyword_arguments["Coulomb_log"].value
+            * constructor_keyword_arguments["n_b"].value
+        )
+
+        expected_limit_values = self.get_limit_value(
+            interaction_type, limit_type, value_test_case
+        )
+
+        for i, (attribute_name, expected_limit_value) in enumerate(
+            zip(self.return_values_to_test, expected_limit_values)
+        ):
+            calculated_limit_value = getattr(value_test_case, attribute_name).value
+
+            if attribute_name != "energy_loss":
+                calculated_limit_value = (
+                    calculated_limit_value / coulomb_density_constant
+                )
+
+            assert np.allclose(
+                calculated_limit_value, expected_limit_value, rtol=0.1, atol=0
+            )
+
+    @pytest.mark.parametrize(
+        "expected_error, constructor_arguments, constructor_keyword_arguments",
+        [
+            (
+                ValueError,
+                (Particle("e-"), Particle("e-")),
+                {
+                    "v_a": 1 * u.cm / u.s,
+                    "T_a": 1 * u.eV,
+                    "T_b": 1 * u.eV,
+                    "n_b": 1 * u.cm**-3,
+                    "Coulomb_log": 1 * u.dimensionless_unscaled,
+                },
+            ),
+            (
+                ValueError,
+                (Particle("e-"), Particle("e-")),
+                {
+                    "T_b": 1 * u.eV,
+                    "n_b": 1 * u.cm**-3,
+                    "Coulomb_log": 1 * u.dimensionless_unscaled,
+                },
+            ),
+        ],
+    )
+    def test_errors(
+        self, expected_error, constructor_arguments, constructor_keyword_arguments
+    ):
+        """Test errors raised in the function body"""
+
+        with pytest.raises(expected_error):
+            CollisionFrequencies(
+                *constructor_arguments, **constructor_keyword_arguments
+            )
+
+    @pytest.mark.parametrize(
+        "constructor_keyword_arguments",
+        [
+            {
+                "test_particle": Particle("e-"),
+                "field_particle": Particle("e-"),
+                "T_a": ones_array * u.eV,
+                "T_b": ones_array * u.eV,
+                "n_b": ones_array * u.cm**-3,
+                "Coulomb_log": ones_array * u.dimensionless_unscaled,
+            },
+        ],
+    )
+    def test_handle_nparrays(self, constructor_keyword_arguments):
```

Comment:
nparray -> ndarray ([technically what the datatype is called](https://numpy.org/doc/stable/reference/arrays.ndarray.html#:~:text=An%20ndarray%20is%20a%20(usually,the%20sizes%20of%20each%20dimension.))